diff --git a/scripts/Training.pl b/scripts/Training.pl
index d10affa..f1ba95b 100644
--- a/scripts/Training.pl
+++ b/scripts/Training.pl
@@ -791,7 +791,7 @@ if ($ENGIN) {
 
       print "Synthesizing a speech waveform from $lab using hts_engine...";
       shell("$hts_engine -or ${dir}/${base}.raw -ot ${dir}/${base}.trace $lab");
-      shell("$SOX -c 1 -s -$SOXOPTION -t raw -r $sr ${dir}/${base}.raw -c 1 -s -$SOXOPTION -t wav -r $sr ${dir}/${base}.wav");
+      shell("$SOX -c 1 -e signed -b 16 -t raw -r $sr ${dir}/${base}.raw -c 1 -e signed -b 16 -t wav -r $sr ${dir}/${base}.wav");
       print "done.\n";
    }
    close(SCP);
@@ -1966,7 +1966,7 @@ sub gen_wave($) {
          $line .= "$DFS -b $lfil | $VOPR -a $gendir/$base.unv | ";
          $line .= "$MGLSADF -m " . ( $ordr{'mgc'} - 1 ) . " -p $fs -a $fw -c $gm $mgc | ";
          $line .= "$X2X +fs -o | ";
-         $line .= "$SOX -c 1 -s -$SOXOPTION -t raw -r $sr - -c 1 -s -$SOXOPTION -t wav -r $sr $gendir/$base.wav";
+         $line .= "$SOX -c 1 -e signed -b 16 -t raw -r $sr - -c 1 -e signed -b 16 -t wav -r $sr $gendir/$base.wav";
          shell($line);
 
          $line = "rm -f $gendir/$base.unv";
